examples/ecs/standalone-task/app.pipecd.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: pipecd.dev/v1beta1
```

Comment:
Should remove those `examples` files from this PR, we can add those files later.